ui/src/data_explorer/containers/DataExplorer.js

```diff
@@ -6,15 +6,17 @@ import _ from 'lodash'
 
 import QueryMaker from '../components/QueryMaker'
 import Visualization from '../components/Visualization'
+import WriteDataForm from '../components/WriteDataForm'
```

Comment:
be the change! -- even though the others here use relative path, let's use absolute path (from alias) for any new code, i.e. `src/data_explorer/components/WriteDataForm`